src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -518,13 +516,11 @@ private boolean isVerifiable(DetailAST methodOrVariableDef) {
      */
     private boolean isContainVerifiableType(DetailAST modifiers) {
         boolean result = false;
-        if (modifiers.getFirstChild() != null) {
```

Comment:
The for loop inside this if is already coded that it ends the loop when this same variable returned is `null`, so it is very safe to remove this.